tensorrt_llm/scaffolding/result.py

```
@@ -1,80 +1,67 @@
+        if isinstance(output, ScaffoldingOutput):
+            self.set_output_streaming(output)
+        # terminate
+        self.set_output_streaming(None)
 
-    async def set_output_async(self, output: GenerationResult):
-        await self.aqueue.put(output)
+    def set_output_streaming(self, output: Union[ScaffoldingOutput, Any]):
+        self.aqueue.put_nowait(output)
 
     def set_task_collections(self, task_collections: Mapping[str,
                                                              "TaskCollection"]):
```

Ruff (0.14.1)

Undefined name `TaskCollection`

(F821)